### ISSUE TITLE
Insecure response when using SuperDevMode

### ISSUE THREAD

**Issue by WTThomas** [gwtproject/gwt#9168]
Hi,

since today I get odd error messages when try to debug my application using Chrome version 44.0.2403.107 m.
![image](https://cloud.githubusercontent.com/assets/3327272/8953028/2a4cc02a-35dc-11e5-9671-6a5a9ec3ec8d.png)
I didn't change anything on my client-server-communication.

This error makes it impsossible to do serious debugging because I can not rely that everything works as it sould in SuperDevMode.

What to do?

Regards
Tom

**Comment by jnehlmeier** [comment]
Can you visit https://127.0.0.1:8888/ directly in Chrome? It will most likely tell you whats wrong with the SSL connection and its certificate. If its a self-signed certificate then Chrome blocks it by default and you need to configure Chrome / your OS to accept that certificate.

Alternative is to simply not use SSL during development.

**Comment by WTThomas** [comment]
Yes, I can visit https://127.0.0.1:8888/ directly in Chrome and view my application.
Chrome is correctly setup to accept my certificate.
Unfortunately it is no option to simply not use SSL during development.

As mentioned above this error message is new. Before It was possible to use SuperDevMode with SSL without such error messages.

**Comment by WTThomas** [comment]
By the way:
These error messages do not appear regularly and every time on every communication.
e.g. Sometimes the png from the screenshot gets loaded sometimes not. Sometimes it's an completely other ressource which does produce an error,...
